src/lib/Components/form-group-component/form-group-component.ts

```diff
@@ -1,29 +1,33 @@
 import {
   Component,
   ContentChildren,
+  ContentChild,
   ElementRef,
   HostBinding,
   Input,
-  QueryList
+  QueryList,
+  AfterContentInit
 } from "@angular/core";
 import { FormControlName } from "@angular/forms";
 import { ErrorMessage } from "../../Models/ErrorMessage";
 import { ErrorMessageService } from "../../Services/error-message.service";
+import { MessagesComponent } from "../messages/messages.component";
 
 @Component({
+  // tslint:disable:component-selector
   selector: ".form-group",
   template: `
     <ng-content></ng-content>
-    <span class="help-block" *ngFor="let message of messages">{{message}}</span>
+    <bfv-messages *ngIf="!helpBlock" [messages]="messages"></bfv-messages>
```

Comment:
The `*ngIf` should now be `*ngIf="!messagesBlock"`

Reply:
Thanks man, good catch. I probably should have waited till the morning when I had fresh eyes, lol.